migrations/env.py

```diff
@@ -55,7 +55,31 @@
 """
 )
 
-register_entities([va_profile_opt_in_out])
+#######################################################################
+# Define and register the stored procedures for VA Profile integration.
+#######################################################################
+
+va_profile_delete_cron_job = PGFunction(
+  schema="public",
+  signature="va_profile_delete_cron_job()",
+  definition="""\
+RETURNS boolean AS
```

Comment:
Your lambda doesn't do anything with a return value.  It seems like you can delete all the logic in this query other than the DELETE statement.